testgrid/config/config.yaml

```diff
@@ -1806,12 +1806,16 @@ dashboards:
   dashboard_tab:
   - name: gke-cvm-1.6-cvm-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-cvm-stable-cvm-master-upgrade-master
+    description: 'Upgrade master only, in gke, from container_vm master-2 to container_vm master'
```

Comment:
argh I'm parsing the wrong thing, basically same concept, but
[ci-kubernetes-e2e-gke] - [cvm-stable] - [cvm-master] - [upgrade-master]